MWSE/TES3Race.cpp

```diff
@@ -9,4 +12,12 @@ namespace TES3 {
 	BodyPart* Race::getBodyPart(bool isFemale, bool isVampire, PartIndex index) {
 		return TES3_Race_getBodyPart(this, isFemale, isVampire, index);
 	}
+
+	char* Race::loadDescription() {
+		return reinterpret_cast<char* (__thiscall*)(Race*)>(TES3_Race_loadDescription)(this);
```

Comment:
See comments about forward declaration of functions in TES3Birthsign.cpp's review.